semantic-python/test/Directive.hs

```diff
@@ -63,7 +63,7 @@ jq = do
 tree :: Trifecta.Parser Directive
 tree = do
   void $ Trifecta.string "# CHECK-TREE: "
-  Tree <$> (Core.Parser.record <|> Core.Parser.comp)
+  Tree <$> (Core.Parser.record <|> Core.Parser.comp <|> Trifecta.parens Core.Parser.core)
```

Comment:
Isn’t `core` the entry point? In which case we probably only need it, and not the other two choices?

Reply:
This is an awful hack to work around a bug that emerges when reading a `CHECK-TREE` statement: if we read it as a function application, the Core parser will start reading the program text (since newlines are not significant in Core).

I am going to fix this in another patch.